main.go

```diff
@@ -102,6 +102,7 @@ func main() {
 
 	// Don't write nginx conf when not in cluster
 	nginx.RunInMockMode = !(kube.RunningInCluster())
+	config.Nginx.RunInMockMode = nginx.RunInMockMode
```

Comment:
Maybe we should get rid of the approch on L104 and always require/use the `config` any place where `RunInMockMode` is checked?  _(Minor nit, should not keep this PR from being merged.)_